build/build.yml

```diff
@@ -105,6 +105,45 @@ steps:
       BuildDropPath: '$(Build.ArtifactStagingDirectory)'
       PackageName: 'MSSQL for VS Code'
 
+  - task: UseDotNet@2
```

Comment:
Do we actually need this? The agents come with stuff already installed so this might not be necessary